dialects/sqlite-3-18/src/main/kotlin/app/cash/sqldelight/dialects/sqlite_3_18/SqliteDialect.kt

```diff
@@ -21,6 +21,9 @@ import timber.log.Timber
  */
 open class SqliteDialect : SqlDelightDialect {
   override val driverType = ClassName("app.cash.sqldelight.db", "SqlDriver")
+  override val asyncDriverType: ClassName? = ClassName("app.cash.sqldelight.async.db", "AsyncSqlDriver")
+  override val asyncCursorType: ClassName? = ClassName("app.cash.sqldelight.async.db", "AsyncSqlCursor")
+  override val asyncPreparedStatementType: ClassName? = ClassName("app.cash.sqldelight.async.db", "AsyncSqlPreparedStatement")
```

Comment:
Should these be promoted as default values and then some of the subclasses dont need these duplicated?